chrome_manifest_v3/views/popupView.js

```
@@ -935,10 +936,12 @@ Gmail2Trello.PopupView.prototype.bindData = function (data) {
       '<option value="none" selected disabled hidden>-</option>' +
       '<option value="d=0 am=0">--</option>';
 
-    $.each(due, function (key, value) {
+    Object.keys(due).forEach(function (key) {
+      var value = due[key];
       if (typeof value === 'object') {
         opt += '<optgroup label="' + key + '">';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)